core/src/main/java/org/infinispan/configuration/cache/CacheMode.java

```diff
@@ -5,6 +5,11 @@
  */
 public enum CacheMode {
    /**
+    * Data is not replicated and certain features are not possible
```

Comment:
I guess 'what we can get without too much effort' just seems a too fluid to me. Maybe at some point we'll be able to add an interceptor stack on the fly, and then we won't need any configuration for this...

@tristantarrant WDYT about making simple-cache a separate cache mode (and configuration element)?